ui-admin/src/api/api.tsx

```diff
@@ -435,8 +452,31 @@ export default {
 
   async fetchEnrolleeChangeRecords(portalShortcode: string, studyShortcode: string, envName: string,
     enrolleeShortcode: string): Promise<DataChangeRecord[]> {
-    const url = `${baseStudyEnvUrl(portalShortcode, studyShortcode, envName)
-    }/enrollees/${enrolleeShortcode}/changeRecords`
+    const url =
+      `${baseStudyEnvUrl(portalShortcode, studyShortcode, envName)}/enrollees/${enrolleeShortcode}/changeRecords`
+    const response = await fetch(url, this.getGetInit())
+    return await this.processJsonResponse(response)
+  },
+
+  async createKitRequest(portalShortcode: string, studyShortcode: string, envName: string,
+                         enrolleeShortcode: string, kitType: string): Promise<string> {
```

Comment:
I want to point out that I don't agree that the suggested indentation is better. I had this:
```
  async createKitRequest(portalShortcode: string, studyShortcode: string, envName: string,
                         enrolleeShortcode: string, kitType: string): Promise<string> {
    const params = new URLSearchParams({ kitType })
    const url =
    ...
```
which makes it clear that the parameter list is split across two lines.

The suggested formatting is:
```
  async createKitRequest(portalShortcode: string, studyShortcode: string, envName: string,
    enrolleeShortcode: string, kitType: string): Promise<string> {
    const params = new URLSearchParams({ kitType })
    const url =
    ...
```
which makes the second line of parameters look like the first line of the body.

An alternative would be one parameter per line:
```
  async createKitRequest(
    portalShortcode: string,
    studyShortcode: string,
    envName: string,
    enrolleeShortcode: string,
    kitType: string
  ): Promise<string> {
    const params = new URLSearchParams({ kitType })
    const url =
    ...
```
but it seems excessive to me to take 7 lines to describe a 5 line function.